src/pages/question.js

```diff
@@ -62,56 +95,83 @@ function Question() {
       user_id: user,
       question_id: questionId,
     });
+    Refresh();
   };
 
   return (
-    <div className="font-audioWide">
+    <div>
       <NavBar />
       <div className="bg-cod-gray absolute h-full w-full flex items-start flex-auto">
-        <div>
-          <h1 className="text-white m-32 text-5xl">{content}</h1>
-          <form className="ml-32 font-[OpenSans]" onSubmit={handleSubmit}>
-            <input
-              name="answer"
-              className="bg-transparent border-none outline-0 placeholder:text-dove-gray max-w-xl text-gray text-2xl w-full"
-              placeholder="Type your answer here"
-            />
-            <div className="bg-white h-1 max-w-xl min-w-min mt-2" />
-            <div className="flex gap-8 mt-8">
-              <button
-                className="bg-cornflower-blue rounded-lg p-4 flex text-white"
-                type="submit"
-              >
-                OK
+        <div className="bg-cover bg-no-repeat bg-galaxy-pattern h-full">
+          <div className="ml-32">
+            <h1 className="text-white text-5xl mt-8">Question of the Week</h1>
+            {answered ? (
+              <div className="text-white mt-36 text-xl mr-24">
+                <h1>
+                  Cool, thank you for sharing. Now you can go check the answers
+                  of other members of the Subvisual Universe!
+                </h1>
                 <img
-                  className="pl-4 pt-1"
-                  src={Vector}
-                  alt="Space Center Logo"
+                  className="max-h-screen basis-2/5 object-cover mt-20 ml-20"
+                  src={Ok}
+                  alt="A background representing the Universe"
                 />
-              </button>
-              <div className="flex">
-                <div className="mt-2 text-2xl text-dove-gray">press </div>
-                <img className="ml-2" src={Enter} alt="Space Center Logo" />
               </div>
-            </div>
+            ) : (
+              <form onSubmit={handleSubmit}>
+                <h1 className="text-white mt-36 text-5xl">{content}</h1>
+                <h1 className="text-white mt-9">
+                  {7 - curr.getDay() + " Days Left"}
```

Comment:
This will only work if the expiry date of a questions is always on a Saturday right? Is that the plan?

Reply:
We want to show 0 days left when we are on sunday, yea.. going to fix it